src/components/ContactsView.js

```diff
@@ -1,23 +1,33 @@
-import { useEffect, useState } from "react"
-import { useParams } from "react-router-dom"
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 
 function ContactsView() {
-  const [contact, setContact] = useState(false)
+	const [contact, setContact] = useState(false);
+	const params = useParams();
 
-  //TODO: Get the contact to load from the params and fetch.
-  //With useEffect, load the contact when params changes
-  //and update contact state
+	useEffect(() => {
+		fetch(`http://localhost:4000/contacts/${params.id}`)
+			.then((response) => response.json())
+			.then((jsonResponse) => setContact(jsonResponse));
```

Comment:
I like the way you've named it jsonResponse. Makes it clearer to me to see what is happening inside that fetch. Niiiice